docs/en/sql-reference/functions/string-functions.md

```diff
@@ -729,6 +729,22 @@ Returns whether string `str` ends with `suffix`.
 endsWith(str, suffix)
 ```
 
+## endsWithUTF8
```

Comment:
Let's add examples when it differs from `endsWith`. And the same for `startsWithUTF8`. Otherwise readers may not see the difference